osf_tests/test_registration_bulk_upload_parser.py

```diff
@@ -79,6 +79,8 @@ def provider_subjects(self):
     @pytest.fixture()
     def registration_provider(self, open_ended_schema, provider_subjects):
         osf_provider = RegistrationProvider.load('osf')
+        osf_provider.default_license = NodeLicense.objects.get(name='No license')
+        osf_provider.licenses_acceptable.add(NodeLicense.objects.get(name='No license'))
```

Comment:
Minor, this fix is fine as long as tests passes. It would be great if we can add the license as it was before the following is removed. For bulk upload, how to handle different type of license (No license, license with or without year and name, etc.) correctly was one issue IIRC.

```
'default_license': 'CC0 1.0 Universal',
'licenses_acceptable': ['CC0 1.0 Universal', 'CC-By Attribution 4.0 International', 'No license'],
```

That being said, this is not necessary if we are adding the above back after `0138_ensure_subjects_and_providers.py` is moved to post-migration signal.